src/views/settings/tabs/RecommendedCapabilitiesTab/context/RecommendedCapabilitiesContext.tsx

```
@@ -0,0 +1,75 @@
+  const resources = useOperatorResources(RECOMMENDED_OPERATOR_PACKAGE_NAMES);
+  const {
+    clusterServiceVersions,
+    filteredPackageManifests,
+    operatorGroups,
+    operatorResourcesLoaded,
+    subscriptions,
+  } = resources;
+
+  const detailsMap = useMemo<RecommendedCapabilityDetailsMap>(
+    () => buildRecommendedDetailsMap(resources, validNamespace),
+    [clusterServiceVersions, filteredPackageManifests, operatorGroups, operatorResourcesLoaded, subscriptions, validNamespace],
```

ESLint

[error] Replace `clusterServiceVersions,·filteredPackageManifests,·operatorGroups,·operatorResourcesLoaded,·subscriptions,·validNamespace` with `⏎······clusterServiceVersions,⏎······filteredPackageManifests,⏎······operatorGroups,⏎······operatorResourcesLoaded,⏎······subscriptions,⏎······validNamespace,⏎····`

(prettier/prettier)